### ISSUE TITLE
How to change window title without recompile flutter_elinux_x11.so with x11 backend?

### ISSUE THREAD

**Issue by eggfly** [sony/flutter-elinux#103]
https://github.com/sony/flutter-embedded-linux/blob/be0a03f403cd78cf8299797818e56c4a1b362259/src/flutter/shell/platform/linux_embedded/window/native_window_x11.cc#L19
I think it is hard coded in source file and it will be compiled into flutter_elinux_x11.so.

**Comment by HidenoriMatsubayashi** [comment]
Right, but elinux for x11 is just for developer's debugging on PC not for embedded devices. So it's not important.
